agot-bg-game-server/src/client/GameSettingsComponent.tsx

```diff
@@ -27,6 +27,7 @@ export default class GameSettingsComponent extends Component<GameSettingsCompone
                 type="checkbox"
```

Comment:
Can you add an id here also when modifying this to allow toggling the checkbox by clicking its label?

Reply:
Indeed, I'll do that!

Reply:
Thx :)